tests/Command/FixMediaContextCommandTest.php

```diff
@@ -142,11 +145,11 @@ public function testExecuteWithEmptyRoot()
 
         $this->pool->expects($this->any())->method('getContexts')->will($this->returnValue(['foo' => $context]));
 
-        $contextModel = $this->getMockBuilder('Sonata\ClassificationBundle\Model\ContextInterface')->getMock();
+        $contextModel = $this->getMockBuilder(ContextInterface::class)->getMock();
 
         $this->contextManger->expects($this->once())->method('findOneBy')->with($this->equalTo(['id' => 'foo']))->will($this->returnValue($contextModel));
 
-        $category = $this->getMockBuilder('Sonata\ClassificationBundle\Model\CategoryInterface')->getMock();
+        $category = $this->getMockBuilder(CategoryInterface::class)->getMock();
```

Comment:
could you replace such lines with `$this->ceateMock()` in a separate commit/PR ?

Reply:
@OskarStark I'll do it in a separate PR tomorrow. I thinks it's too much for this one.